common/transport/amqp/package.json

```diff
@@ -7,14 +7,12 @@
   "main": "main.js",
   "typings": "index.d.ts",
   "dependencies": {
-    "amqp10": "^3.6.0",
-    "amqp10-transport-ws": "^0.0.5",
+    "azure-iot-common": "1.5.0",
```

Comment:
>1.5.0", [](start = 25, length = 7)

reverting? should be 1.5.2 apparently

Reply:
My bad on the merge.

---
In reply to: [191904127](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191904127) [](ancestors = 191904127)